app/src/main/res/values/strings.xml

```diff
@@ -87,185 +87,186 @@
     <string name="history_no_games">You haven\'t played yet</string>
     <string name="history_item_time">Time: %1$s</string>
     <string name="history_item_id">Game ID: %1$d</string>
-    <string name="history_item_can_continue">You can continue this game</string>
+    <string name="history_item_can_continue">Can be continued</string>
     <string name="saved_game_screen_title">Game #%1$d</string>
     <string name="saved_game_current">Current</string>
     <string name="saved_game_initial">Initial</string>
     <string name="saved_game_progress">Progress %1$d/%2$d</string>
     <string name="saved_game_progress_percentage">Progress %1$d%%</string>
-    <string name="empty_screen_something_went_wrong">Oops, something went wrong…</string>
+    <string name="empty_screen_something_went_wrong">Something went wrong…</string>
     <string name="saved_game_difficulty">Difficulty: %1$s</string>
     <string name="saved_game_type">Type: %1$s</string>
     <string name="saved_game_time">Time: %1$s</string>
-    <string name="saved_game_completed">The game was completed</string>
-    <string name="saved_game_give_up">The game was not completed, you gave up</string>
+    <string name="saved_game_completed">Completed</string>
+    <string name="saved_game_give_up">You have given up.</string>
     <string name="saved_game_in_progress">In progress</string>
-    <string name="saved_game_mistakes_limit">The game has not been completed, you have exceeded the mistakes limit</string>
+    <string name="saved_game_mistakes_limit">Mistake-limit exceeded.</string>
     <string name="learn_screen_title">Learn</string>
     <string name="learn_tab_sudoku">Sudoku</string>
     <string name="learn_tab_app">App</string>
     <string name="learn_app_toolbar">Toolbar</string>
     <string name="learn_app_toolbar_desc">A panel under the keyboard</string>
-    <string name="learn_app_toolbar_notes_menu">You can press and hold the Notes button to open the Notes menu, where you can clear or compute notes and toggle their visibility</string>
-    <string name="learn_sudoku_rules">Sudoku rules</string>
-    <string name="sudoku_rules_mistakes">There is a mistake here, can you spot it? Maybe even two</string>
+    <string name="learn_app_toolbar_notes_menu">Press and hold the \"Notes\" button to compute, clear, or hide/show notes.</string>
+    <string name="learn_sudoku_rules">Rules</string>
+    <string name="sudoku_rules_mistakes">Spot the mistake(s) here</string>
     <string name="sudoku_rules_mistakes_highlight">Highlight mistakes</string>
-    <string name="sudoku_rules_mistakes_explanation">The first mistake is two 2s in the sixth box. Remember the rule that there can\'t be double values in a box? The second mistake is the number 6 in row 2, column 8 (R2C8), because there is already a 6 in that column in R5C8</string>
+    <string name="sudoku_rules_mistakes_explanation">2 can not be repeated within a (3x3) box? There can only be one 6 in each row.</string>
```

Comment:
One button to highlight each of these, with the border of the respective box and row highlighted works better(?).
The notation can be explained in a different category, as it isn't crucial for beginners.